packages/features/ee/organizations/context/provider.ts

```diff
@@ -10,10 +10,15 @@ import type { teamMetadataSchema } from "@calcom/prisma/zod-utils";
  */
 export type OrganizationBranding =
   | ({
+      /** 1 */
       id: number;
+      /** Acme */
       name?: string;
+      /** acme */
       slug: string;
+      /** https://acme.cal.com */
       fullDomain: string;
+      /** cal.com */
```

Comment:
These comments are here to help visualize what they represent when hovering their usage, like this:
![image](https://github.com/calcom/cal.com/assets/467258/4dcb2554-57b8-40d4-bc60-801a7ce43f18)
![image](https://github.com/calcom/cal.com/assets/467258/d2038c26-4eee-48a6-8ad0-55d11e3ed245)